index.bs

```diff
@@ -187,6 +187,10 @@ After several weeks, she has developed a tiny DNN model that can even run on
 CPU. In order to accommodate CPU execution, she modifies the application
 so that the application loads the tiny model in the case of CPU-only devices.
 
+### Operation Level Execution ### {#usecase-op-level-exec}
+
+A JavaScript ML framework is responsible for loading, interpreting and executing a ML model. During the model execution phase, the framework iterates through the operations of the model and executes each operation on the hardware device, like CPU or GPU. To avoid the unnecessary data copying cross devices, the framework selects the same device to execute the operations. For a compute intensive operation, such as convolution 2D or matrix multiplication, the framework uses WebNN API to execute it with the ML-specific acceleration available on that selected device.
```

Comment:
"an ML graph."

Reply:
Thanks. May I know which word you suggest to replace with "an ML graph"?

Reply:
just a wording suggestion: "a ML model" -> "an ML graph"

Reply:
Thanks for the suggestion. I think a ML model here refers to what the JS framework handles, e.g. TensorFlow models of TensorFlow.js, ONNX models of ONNX.js. I don't want to blur it with ML graph of WebNN that is a framework agnostic in-memory representation. Does it make sense?